src/middlewares/sendEmail.js

```diff
@@ -26,6 +26,7 @@ export const sendVerificationEmail = async (req, res, next) => {
     }
 
   } catch (error) {
+    next(error);
```

Comment:
@edgar929 you can pass the error object then in the errorHandler function, you can display whatever property of error.